src/weatherreport/src/weatherreport_getopt.erl

```diff
@@ -0,0 +1,621 @@
+%%%-------------------------------------------------------------------
+%%% @author Juan Jose Comellas <juanjo@comellas.org>
+%%% @copyright (C) 2009 Juan Jose Comellas
+%%% @doc Parses command line options with a format similar to that of GNU getopt.
+%%% @end
+%%%
+%%% This source file is subject to the New BSD License. You should have received
+%%% a copy of the New BSD license with this software. If not, it can be
+%%% retrieved from: http://www.opensource.org/licenses/bsd-license.php
```

Comment:
@rnewson @janl I think this means we need to add this file to `/NOTICE`, yes?

Reply:
I've added it just in case.